deepgram/transcription.py

```diff
@@ -102,6 +170,16 @@ async def _start(self) -> None:
         )
 
     async def _receiver(self) -> None:
+        """
+        The _receiver function is a coroutine that receives messages from the socket and puts them in a queue.
+        It is started by calling start_receiver() on an instance of AsyncSocket. It runs until the socket is closed,
+        or until an exception occurs.
+        
+        :param self: Used to Access the attributes of the class.
```

Comment:
Please remove this line from docstring `:param self: Used to Access the attributes of the class.`